### PR TITLE
⭐️ arista provider

### PR DESCRIPTION
**Pull request by chris-rock**
- adds arista to v9 
- improves the default values for resources

```
cnquery shell arista admin@192.168.178.154 --ask-pass
```

```coffeescript
cnquery> arista.eos { version systemConfig} 
arista.eos: {
  version: {
    architecture: "x86_64"
    bootupTimestamp: 1633245562.000000
    configMacAddress: "00:00:00:00:00:00"
    hardwareRevision: ""
    hwMacAddress: "00:00:00:00:00:00"
    internalBuildId: "37a9ea6d-2c81-487d-9c86-579348318c0a"
    internalVersion: "4.25.1F-20005270.4251F"
    isIntlVersion: false
    memFree: 880184.000000
    memTotal: 2012348.000000
    mfgName: ""
    modelName: "vEOS"
    serialNumber: ""
    systemMacAddress: "00:0c:29:ed:72:0e"
    uptime: 275227.470000
    version: "4.25.1F"
  }
  systemConfig: {
    hostname: "localhost"
    iprouting: "false"
  }
}
cnquery> arista.eos.ipInterfaces
arista.eos.ipInterfaces: [
  0: arista.eos.ipInterface name="defaults"
  1: arista.eos.ipInterface name="Management1"
]
cnquery> arista.eos.interfaces
arista.eos.interfaces: [
  0: arista.eos.interface name="Ethernet2"
  1: arista.eos.interface name="Management1"
  2: arista.eos.interface name="Ethernet1"
]
```

### CODE REVIEW

File: providers/arista/provider/provider.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+		}
+
+		x, err := url.Parse(target)
+		if err != nil {
+			return nil, errors.New("incorrect format of target, please use user@host:port")
+		}
+
+		user = x.User.Username()
+		conn.Host = x.Hostname()
+		conn.Path = x.Path
```
Comment (czunker) [inline providers/arista/provider/provider.go:59]: Is this intended?

The error above states "please use user@host:port". Nothing about a path. It is also not used in `NewAristaConnection`.

**Review by czunker — COMMENTED** [review]
I cannot test it. I've no access to Arista equipment.

File: providers/arista/provider/provider.go (hunk 1 above)
Comment (chris-rock) [reply providers/arista/provider/provider.go:59]: no, that is a copy paste bug :-)

**Review by arlimus — APPROVED** [review]
Arista is back!

**Pull request merged**